op-node/flags/flags.go

```diff
@@ -100,6 +100,12 @@ var (
 		Required: false,
 		Hidden:   true,
 	}
+	L1RPCMaxConcurrentRequests = &cli.IntFlag{
+		Name:    "l1.max-concurrent-rpc-requests",
+		Usage:   "Maximum number of concurrent RPC requests to make to the L1 RPC provider.",
+		EnvVars: prefixEnvVars("L1_MAX_CONCURRENT_RPC_REQUESTS"),
+		Value:   10,
+	}
```

Comment:
I'm wondering if we want to shorten this a bit, similarly to other flags, e.g.
```suggestion
	L1RPCMaxConcurrency = &cli.IntFlag{
		Name:    "l1.max-concurrency",
		Usage:   "Maximum number of concurrent RPC requests to make to the L1 RPC provider.",
		EnvVars: prefixEnvVars("L1_MAX_CONCURRENCY"),
		Value:   10,
	}
```
this would then also apply to config field names, variables etc further below.